care/facility/models/post_covid_data.py

```diff
@@ -0,0 +1,206 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField, ArrayField
+from care.facility.models.base import BaseModel
+from care.facility.models.patient_consultation import PatientConsultation
+import enum
+
+
+class PostCovidTime(enum.Enum):
+    UNDER_THREE_WEEKS = 1
+    THREE_TO_TWELVE = 2
+    OVER_TWELVE_WEEKS = 3
+
+
+PostCovidTimeChoices = [(e.value, e.name) for e in PostCovidTime]
+
+
+class CovidCategory(enum.Enum):
+    MILD = 1
+    MODERATE = 2
+    SEVERE = 3
+
+
+CovidCategoryChoices = [(e.value, e.name) for e in CovidCategory]
+
+
+class OxygenType(enum.Enum):
+    NASAL_PRONGS = 1
+    FACE_MASK = 2
+    NRBM = 3
+    FNO = 4
+
+
+OxygenTypeChoices = [(e.value, e.name) for e in OxygenType]
+
+
+class AnticoagulantModeOfTransmission(enum.Enum):
+    IV = 1
+    ORAL = 2
+
+
+AnticoagulantModeOfTransmissionChoices = [
+    (e.value, e.name) for e in AnticoagulantModeOfTransmission
+]
+
+
+class PostCovidData(BaseModel):
+    consultation = models.OneToOneField(
+        PatientConsultation,
+        on_delete=models.PROTECT,
+        null=False
+    )
+    patient = models.ForeignKey(
+        "PatientRegistration",
+        on_delete=models.PROTECT,
+        null=True
+    )
+    date_of_onset_symptoms = models.DateField(
+        default=None,
+        null=True,
+        verbose_name="Date of first appearance of symptoms"
+    )
+    post_covid_time = models.IntegerField(
+        choices=PostCovidTimeChoices,
+        default=1,
```

Comment:
Becomes difficult to understand and hence debug. Use `PostCovidTime.UNDER_THREE_WEEKS`. I had difficulty finding the value for `1` while review, we cannot expect the next developers to know the number values and use it.